sampleConfig.wdl

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2018 Sequencing Analysis Support Core - Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task SampleConfig {
+    File jar
+    Array[File]+ inputFiles
+    String? sample
+    String? library
+    String? readgroup
+    String? jsonOutputPath
+    String? tsvOutputPath
+
+    command {
+        mkdir -p . $(dirname ${jsonOutputPath}) $(dirname ${tsvOutputPath})
+        java -jar ${jar} \
+        -i ${sep="-i " inputFiles} \
+        ${"--sample " + sample} \
+        ${"--library " + library} \
+        ${"--readgroup " + readgroup} \
+        ${"--jsonOutput " + jsonOutputPath} \
+        ${"--tsvOutput " + tsvOutputPath}
+    }
+
+    output {
+        Array[String] keys = read_lines(stdout())
+        File? jsonOutput = jsonOutputPath
+        File? tsvOutput = tsvOutputPath
+        Object values = if (defined(tsvOutput) && size(tsvOutput) > 0) then read_map(tsvOutput) else { "": "" }
+    }
+}
+
+task DownloadSampleConfig {
+    String? release = "0.1"
```

Comment:
Maybe this download task we should not use anymore, it's not explicit enough. (sorry my fault)